frontend/src/components/forms/CreateLog.tsx

```diff
@@ -183,59 +263,48 @@ const CreateLog = () => {
   };
 
   const handleSubmit = () => {
-    // log the state values for testing
-    // console.log({
-    //     employee,
-    //     date,
-    //     time,
-    //     building,
-    //     resident,
-    //     tags,
-    //     attnTo,
-    //     notes,
-    //     flagged,
-    // });
-
     // Update error states
-    setEmployeeError(employee === "");
+    setEmployeeError(employee.label === "" && employee.value === -1);
```

Comment:
Nit: I believe an empty string is falsy and you can just do `!employee.label` (also below on line 276)